### PR TITLE
Add language icon to team stream

### PR DESCRIPTION
**Pull request by bradhvr**
I added the language icon to the top right of the team stream solutions, per issue #3020.

### CODE REVIEW

**Comment by Insti** [comment]
Great :heart: Can you add before/after screenshots to this PR please.
(I know you've done some in the linked issue but it makes evaluating the PR easier if the changes are clearly visible here when reviewing.)

**Comment by bhoover85** [comment]
Sure, here you go...

**Before**
2016-08-21_11h46_18

**After**
2016-08-21_11h45_43

**Comment by Insti** [comment]
Will this also cause the icon to appear on regular language streams?
In which case all the language icons will be the same.

(It's probably fine if it does, but it would be nice to have the consideration documented.)

**Comment by bhoover85** [comment]
You mean if someone filters by one of the languages? Yes it will appear on those as well.

2016-08-21_12h33_44

**Comment by Insti** [comment]
Also on the regular language tracks that aren't part of a team stream, for example: http://exercism.io/tracks/ruby/exercises

**Comment by bhoover85** [comment]
Oh I see... yes it does.

2016-08-21_12h53_55

File: app/views/stream/_main.erb [hunk 1]
```diff
@@ -17,6 +17,7 @@
               <h4>
                 <%= exercise.problem.name %>
                 <span class="profile"> <%= exercise.username %> </span>
+                <span class="track-icon"><%= track_icon(exercise.problem.track_id, 20) %></span>
```
Comment (radar) [inline app/views/stream/_main.erb]: Is it possible to get some alt-text for this icon too?

Comment (Insti) [inline app/views/stream/_main.erb]: @radar that will require changing the track_icon view helper and probably belongs in a different PR, will you create an issue for it please?

**Comment by kytrinyx** [comment]
I merged it manually in 0de777dd1d83152005e9b31b9ec6781bc31cb685

**Comment by kytrinyx** [comment]
Thank you so much @bradhvr!

**Comment by bradhvr** [comment]
@kytrinyx, I think my CSS change may inadvertently have gotten dropped in the merge. It looks like the compiled `application.css` is missing my change in `inbox.scss`:

```
.track-icon {
  float: right;
}
```

Here's what I see in my dev environment:
image

What would be the best way to fix this?

**Comment by kytrinyx** [comment]
I'll recompile locally and see if it gets added.

**Comment by kytrinyx** [comment]
I think that worked. Thanks for catching that.

**Comment by bradhvr** [comment]
Great, thanks!
